### PR TITLE
Refactor yes/no toggle component

### PR DESCRIPTION
**Pull request by NeOMakinG**
Related with https://github.com/PrestaShop/PrestaShop/issues/16604

Job is dont but https://github.com/PrestaShop/prestashop-ui-kit/pull/73 needs to be merged before so we can lint and change colors to vars

### CODE REVIEW

**Comment by eternoendless** [comment]
Needs a rebase

File: scss/_ps-switch.scss [hunk 1]
```diff
@@ -1,77 +1,149 @@
...
-    text-align: center;
-    text-transform: uppercase;
-    vertical-align: middle;
-    cursor: pointer;
-    @include transition(color .2s ease-out);
-  }
+    label {
+        position: absolute;
+        left: 0;
+        padding-left: 45px;
```
Comment (PierreRambaud) [inline scss/_ps-switch.scss]: use rem

File: scss/_ps-switch.scss [hunk 2]
```diff
@@ -1,77 +1,149 @@
...
+        }
+    }
+
+    &.ps-switch-sm {
+      min-width: 6.250rem;
+      font-size: $font-size-sm;
+      height: 16px;
+
+        label {
+            padding-left: 40px;
```
Comment (PierreRambaud) [inline scss/_ps-switch.scss]: use rem

File: scss/_ps-switch.scss [hunk 3]
```diff
@@ -1,77 +1,149 @@
...
+            }
+        }
+    }
+
+    &.ps-switch-lg {
+        font-size: $font-size-lg;
+        height: 30px;
+
+        label {
+            padding-left: 65px;
```
Comment (PierreRambaud) [inline scss/_ps-switch.scss]: use rem

**Comment by matks** [comment]
Might be related to https://github.com/PrestaShop/PrestaShop/pull/13458

**Comment by sarahdib** [comment]
The PR has not changed anything :

After this PR
Capture d’écran 2020-03-24 à 09 01 39

**Comment by PierreRambaud** [comment]
Thanks @NeOMakinG

**Pull request merged**